xc/device.ml

```diff
@@ -1436,6 +1437,10 @@ module Dm = struct
 
   let max_emulated_nics = 8 (** Should be <= the hardcoded maximum number of emulated NICs *)
 
+  let (qmp_event_pipe_r, qmp_event_pipe_w) = Unix.pipe ()
```

Comment:
qmp_event_pipe_r -> QMP_Event.wakeup_fd_r ,same for _w (see below for QMP_Event)

Reply:
Thanks, now move this part to `module QMP_Event`